README.md

```diff
@@ -79,6 +79,15 @@ from django.core.urlresolvers import reverse_lazy
 LOGOUT_URL = reverse_lazy('loginas-logout')
 ```
 
+Additionally you can also specify the redirect url for logout default is `settings.LOGIN_REDIRECT_URL`
```

Comment:
Can you clean up the wording a bit? Like so:

Additionally, you can specify the redirect url for logout (the default is `settings.LOGIN_REDIRECT_URL`).